src/Components/Programs/Timeline.js

```diff
@@ -0,0 +1,99 @@
+import React, {useEffect} from 'react';
+import {ScrollView,Text,View,TouchableWithoutFeedback} from 'react-native';
+import {Event,Line,Date,Marker,Fade,Stroke} from './styles';
+import $ from 'jquery';
+import {ArrowLeftOutlined ,ArrowRightOutlined} from '@ant-design/icons'
+import dayjs from 'dayjs'
+const events=[
+    {
+        event:"GSoC'20",
+        date:[[0,1],[0,7],[0,8],[0,12],[0,18],[0,20],[0,31],[1,10],[1,20]],
+        color:'#ffdd30'
+    },
+    {
+        event:"Rails Girls'20",
+        date:[[0,5],[0,7],[0,9],[0,22],[0,28],[0,31],[1,8],[1,10]],
+        color:'#039eff'
+    },
+]
+const months = [[0,"January"],[1,"February"],[2,"March"],[3,"April"],[4,"May"],[5,"June"],[6,"July"],[7,"August"],[8,"September"],[9,"October"],[10,"November"],[11,"December"]];
+function Timeline() {
+    var p = dayjs().date();
+    useEffect(() => {
+        var scroll = $('.css-view-1dbjc4n');
+        scroll.scrollLeft(p);
+    })
+    const scrollLeft=()=>{
+        var scroll = $('.css-view-1dbjc4n');
+        scroll.scrollLeft(p-=35);
+        if(p<0)
+        p=0;
+    }
+    const scrollRight=()=>{
+        var scroll = $('.css-view-1dbjc4n');
+        scroll.scrollLeft(p+=35);
+    }
+  return (
+      <>
```

Comment:
what this line do?

Reply:
It handles the onClick events . When the user clicks the past or future arrows, it scrolls the timeline accordingly. That long className targets the scrollview.